src/main.rs

```diff
@@ -0,0 +1,75 @@
+mod commands;
+mod event_handler;
+
+use dotenv::dotenv;
+use std::env;
+
+use color_eyre::eyre::{Report, Result};
+use poise::serenity_prelude::{self as serenity, ActivityData, GatewayIntents};
+
+#[derive(Debug)]
+pub struct Data {} // User data, which is stored and accessible in all command invocations
+
+pub type Context<'a> = poise::Context<'a, Data, Report>;
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    env_logger::init();
+
+    // Load the .env file
+    dotenv().ok();
+
+    // Enable color_eyre beacuse error handling ig
+    color_eyre::install()?;
```

Comment:
Least helpful comments ive ever seen lol

Reply:
it helps me •`ᴖ´•

Reply:
What else could color_eyre::install do besides install color_eyre and what else could dotenv() be doing besides loading the dot env file 😭